components/button/button-icon.js

```diff
@@ -110,18 +110,18 @@ class ButtonIcon extends ButtonMixin(VisibleOnAncestorMixin(RtlMixin(LitElement)
 				aria-expanded="${ifDefined(this.ariaExpanded)}"
 				aria-haspopup="${ifDefined(this.ariaHaspopup)}"
 				aria-label="${this.ariaLabel ? this.ariaLabel : ifDefined(this.text)}"
-				?autofocus="${ifDefined(this.autofocus)}"
+				?autofocus="${this.autofocus}"
 				class="d2l-label-text"
 				?disabled="${this.disabled}"
 				form="${ifDefined(this.form)}"
 				formaction="${ifDefined(this.formaction)}"
 				formenctype="${ifDefined(this.formenctype)}"
 				formmethod="${ifDefined(this.formmethod)}"
-				formnovalidate="${ifDefined(this.formnovalidate)}"
+				?formnovalidate="${this.formnovalidate}"
```

Comment:
`formnovalidate` is a boolean attribute type but we weren't using a boolean binding. And then the `ifDefined` wasn't needed.